apps/server/src/modules/deletion/console/deletion-console.app.ts

```diff
@@ -0,0 +1,30 @@
+/* istanbul ignore file */
+import { BootstrapConsole } from 'nestjs-console';
+import { DeletionConsoleModule } from './deletion-console.module';
+
+const bootstrap = new BootstrapConsole({
+	module: DeletionConsoleModule,
+	useDecorators: true,
+});
+
+bootstrap
+	.init()
+	.then(async (app) => {
+		// eslint-disable-next-line promise/always-return
+		try {
+			await app.init();
+			await bootstrap.boot();
+			await app.close();
+		} catch (err) {
+			await app.close();
+
+			// eslint-disable-next-line no-console
+			console.error(err);
+			process.exitCode = 1;
+		}
+	})
+	.catch((err) => {
+		// eslint-disable-next-line no-console
+		console.error(err);
+		process.exitCode = 1;
+	});
```

Comment:
Please rewrite it to try catch sturctur. And i think you should move this file to /server/src/apps/

Reply:
Good point, will do that.

Reply:
I've moved the console app file to the `apps` dir, but I was unable to rewrite it to just try/catch as I was always falling into the
> TS1378: Top-level  await  expressions are only allowed when the  module  option is set to  es2022 ,  esnext ,  system ,  node16 , or  nodenext , and the  target  option is set to  es2017  or higher. 

Do you want me to add an exception to be able use the await in the top-level of the script anyway? Or maybe could you give me some hint or some draft of an example how you'd like to have it done?

Reply:
After a hint from @SevenWaysDP , the solution was modified in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4521/commits/cfa6ac9e14968dcf6ec3b833e02fb7eb5a27c7a7?diff=split&w=0 .